### PR TITLE
3.1 - Add filter options and links

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 1]
```diff
@@ -73,6 +78,23 @@ You can also filter on completeness in order to see the records that are complet
...
+
+_Since 3.1_, we added the ability to filter on attributes types single/multiple options or reference entity single/multiple links in the records grid. It allows an exact search to find the records with a specific value (option value or link to a record).
+
+For examples:
+- If a "Brand" reference entity has a "Designers" attribute linked to a "Designer" reference entity, you can search for all brands with the designer "Ron Arad".
+- If a "Brand" reference entity has a "Country" attribute with a list of options, you can search for brands with the country "Italy".
+
+![Records grid](../img/Records_filter_options.png)
+
+The operator is an "in list" operator. If you are looking for the brand with the designer "Philippe Starck" OR "Ron Arad", the brands with the designer "Philippe Starck" OR the designer "Ron Arad" will be displayed.
```
Comment (NolwennP) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]:
```suggestion
The operator is an `in list` operator. If you are looking for the brand with the designer "Philippe Starck" OR "Ron Arad", the brands with the designer "Philippe Starck" OR the designer "Ron Arad" will be displayed.
```

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 2]
```diff
@@ -73,6 +78,23 @@ You can also filter on completeness in order to see the records that are complet
 To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
 :::
 
+## Filter on options and reference entity links (_since 3.1_)
+
+_Since 3.1_, we added the ability to filter on attributes types single/multiple options or reference entity single/multiple links in the records grid. It allows an exact search to find the records with a specific value (option value or link to a record).
+
+For examples:
```
Comment (morganen) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]: Je pense qu'il n'y a pas de "s" quand on met "for" avant.

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 3]
```diff
@@ -43,7 +43,12 @@ The records grid is quite different than the products' one. Thereafter are the s
 - Finally, the grid can display up to 500 results using an **infinite scroll**. If you have more results, please filter your search. When reaching this limit, the grid displays the message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*
 
 ::: info
-In the records grid, you will see only certain types of attributes: code, text and image.
+In the records grid in 3.0, you will see only certain types of attributes: code, text and image.  
+_Since 3.1_, the values of the attributes types single/multiple options and reference entity single/multiple links are also displayed in the records grid.
```
Comment (LaureBro) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]:
```suggestion
_Since the 3.1 version_, the values of the attributes types single/multiple options and reference entity single/multiple links can also be displayed in the records grid.
```

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 4]
```diff
@@ -73,6 +78,23 @@ You can also filter on completeness in order to see the records that are complet
 To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
 :::
 
+## Filter on options and reference entity links (_since 3.1_)
+
+_Since 3.1_, we added the ability to filter on attributes types single/multiple options or reference entity single/multiple links in the records grid. It allows an exact search to find the records with a specific value (option value or link to a record).
```
Comment (LaureBro) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]:
```suggestion
_Since the 3.1 version_, we added the possibility to filter on single/multiple options or reference entity single/multiple links attributes types in the records grid. This filter allows an exact search on an reference entity option value or on a reference entity link. 
```

File: content/md/products-super-power/enrich-your-reference-entity-records.md (hunk 2 above)
Comment (LaureBro) [reply content/md/products-super-power/enrich-your-reference-entity-records.md]: you can use "for instance", or "e.g."

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 5]
```diff
@@ -73,6 +78,23 @@ You can also filter on completeness in order to see the records that are complet
...
+
+For examples:
+- If a "Brand" reference entity has a "Designers" attribute linked to a "Designer" reference entity, you can search for all brands with the designer "Ron Arad".
+- If a "Brand" reference entity has a "Country" attribute with a list of options, you can search for brands with the country "Italy".
+
+![Records grid](../img/Records_filter_options.png)
+
+The operator is an "in list" operator. If you are looking for the brand with the designer "Philippe Starck" OR "Ron Arad", the brands with the designer "Philippe Starck" OR the designer "Ron Arad" will be displayed.
+
+You can combine the filters. For example, you can search for all brands with the country "Italy" AND the designer "Ron Arad".
```
Comment (LaureBro) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]:
```suggestion
You can combine filters. For instance, you can search on all brands with the country "Italy" AND the designer "Ron Arad".
```

Comment (LaureBro) [reply content/md/products-super-power/enrich-your-reference-entity-records.md]: this example is not very clear to me :(

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 6]
```diff
@@ -43,7 +43,12 @@ The records grid is quite different than the products' one. Thereafter are the s
 - Finally, the grid can display up to 500 results using an **infinite scroll**. If you have more results, please filter your search. When reaching this limit, the grid displays the message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*
 
 ::: info
-In the records grid, you will see only certain types of attributes: code, text and image.
+In the records grid in 3.0, you will see only certain types of attributes: code, text and image.  
+_Since 3.1_, the values of the attributes types single/multiple options and reference entity single/multiple links are also displayed in the records grid.
+:::
+
+::: info
+The order of the columns in the records grid is "Image, Label, Code, Complete", the other attributes are then ordered according to their order of creation in the "Attributes" tab.
```
Comment (LaureBro) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]: The columns' order in the records grid is frozen, it cannot be changed. By default, the grid's column order is `Image`, `Label`, `Code`, `Complete`. If you add other attributes, under the `Attributes` tab, the grid will display them according to their order of creation.

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 7]
```diff
@@ -73,6 +78,23 @@ You can also filter on completeness in order to see the records that are complet
 To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
 :::
 
+## Filter on options and reference entity links (_since 3.1_)
+
+_Since the 3.1 version_, we added the possibility to filter on single/multiple options or reference entity single/multiple links attributes types in the records grid. This filter allows an exact search on an reference entity option value or on a reference entity link. 
+
+For examples:
```
Comment (LaureBro) [inline content/md/products-super-power/enrich-your-reference-entity-records.md]:
```suggestion
For instance:
```

File: content/md/products-super-power/enrich-your-reference-entity-records.md [hunk 8]
```diff
@@ -43,7 +43,12 @@ The records grid is quite different than the products' one. Thereafter are the s
 - Finally, the grid can display up to 500 results using an **infinite scroll**. If you have more results, please filter your search. When reaching this limit, the grid displays the message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*
```
[the remark this answers is not in the document]
Comment (LaureBro) [reply content/md/products-super-power/enrich-your-reference-entity-records.md]:
```suggestion
- Finally, the grid comes with an **infinite scroll** that allows you to display up to 500 results. If you reach this limit, the grid will show the following message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*
```

**Pull request merged**